releases/release-1.22/release-notes/maps/pr-101560-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 101560
+releasenote:
+  do_not_publish: true
```

Comment:
I do think this PR needs to be published and to have a proper release note. I see the owner did not set one, that does not mean we should not try to investigate and set a good explanation. 
Example:

Fixes a panic in `kube-scheduler` when accessing not existing pods while using `DefaultPreemption ` plugin.

Reply:
Done